db/upgrade.php

```diff
@@ -235,5 +235,25 @@ function xmldb_customcert_upgrade($oldversion) {
         upgrade_mod_savepoint(true, 2022112804, 'customcert');
     }
 
+    if ($oldversion < 2022112809) {
+        $elements = $DB->get_records('customcert_elements', ['element' => 'date']);
+
+        foreach ($elements as $element) {
+            $data = json_decode($element->data);
+
+            // If dateitem is between CUSTOMCERT_DATE_EXPIRY_ONE and
+            // CUSTOMCERT_DATE_EXPIRY_FIVE.
+            if ((intval($data->dateitem) <= -8) && (intval($data->dateitem) >= -12)) {
+                $data->startfrom = 'award';
```

Comment:
Should this be a string in the language folder?

Reply:
Sorry, if I actually bothered looking at the code in more detail I would have realised that. Using a custom string would break it. Ignore me.